app/post.go

```diff
@@ -577,6 +577,58 @@ func (a *App) GetPostsAroundPost(postId, channelId string, offset, limit int, be
 	}
 }
 
+func (a *App) GetPostAfter(channelId string, time int64) (*model.Post, *model.AppError) {
```

Comment:
Why use timestamps here instead of IDs? 0/5 between the two, but I initially expected that this to take a post ID

Reply:
We're using timestamp at store to get post after, and since such timestamp is available before calling this function, we don't need to call get post just in case post ID is passed in here.
Maybe change function name, like `GetPostAfterTime`?

Reply:
Yeah just rename the function name to be more explicit, but it makes sense to query per timestamp

Reply:
Rename the function to `GetPostBeforeTime` and `GetPostAfterTime`.